rotkehlchen/types.py

```diff
@@ -948,6 +948,7 @@ class CacheType(Enum):
     SPAM_ASSET_FALSE_POSITIVE = auto()  # assets that shouldn't be marked as spam automatically
     COINLIST = auto()  # coinlist / all coins cache for various oracles
     AIRDROPS_METADATA = auto()  # airdrops index fetched from rotki/data repo
+    AIRDROPS_CSV_HASH = auto()  # hash of airdrops csv file
```

Comment:
try to keep the names a bit shorter since they take space in the DB as part of the key with the current design

```suggestion
    AIRDROPS_HASH = auto()  # hash of airdrops csv file
```